### DISCUSSION TITLE
View-Only mode for SuperEditor

### DISCUSSION THREAD

**Discussion by MenuItem207** [Flutter-Bounty-Hunters/super_editor#734] [Q&A] [answered]
I've been trying to use this package for a note-taking app that includes multiple rich text sections. I've noticed that when using multiple SuperEditor widgets in a ListView, multiple carets show up. Assigning multiple FocusNodes and calling node.unfocus() doesn't seem to work either. So my question is, is there a view-only SuperEditor widget that disables the caret? I've looked through the repo and can't seem to locate one!

Sorry if it actually exists and I'm just asking a obvious question :) Thank you for taking the time to read this!

**Comment by matthew-carroll** [comment, marked as answer]
We have an upcoming "read-only" milestone here: https://github.com/superlistapp/super_editor/discussions/303

However, the issue of multiple `SuperEditor`s at the same time is it's own issue. Please file a ticket with a minimal reproduction of the problem and we'll take a look at it when we can.

**Reply by MenuItem207** [reply]
Created the issue https://github.com/superlistapp/super_editor/issues/738.
